import { LedgerCustom, LedgerError, LedgerErrorReason } from '../common';
/**
 * Error thrown when the bridge reported fail in prepare phase.
 * When ledger core adds a signature with this reason it will set
 * failId to the reason received in failed signature from a bridge.
 */
export declare class CoreBridgePrepareFailed extends LedgerError {
    /**
     *
     * @param detail error detail - message
     * @param custom custom data record to attach
     *               additional info about the error
     */
    constructor(detail: string, custom?: LedgerCustom);
    static REASON: LedgerErrorReason;
}
